Lib/messages.rb

```diff
@@ -0,0 +1,42 @@
+require 'pg'
+
+class Messages
+
+  attr_reader :id, :content, :created_at, :name
+
+  def initialize(id, content, created_at, name)
+    @id = id
+    @content = content
+    @created_at = created_at
+    @name = name
+  end
+
+  def self.add(content, name)
+    database_connection.exec("INSERT INTO messages(content,name) 
+      VALUES('#{content}','#{name}')")
+  end
+
+  def self.show
```

Comment:
conventionally, a method like this which retrieves every example of a particular resource would be called .all